src/main/java/frc/robot/Ports.java

```diff
@@ -1,5 +1,8 @@
 package frc.robot;
 
 public final class Ports {
-
+public class Gripper{
+    public static final int LEFT_SOLENOID = 0;
+    public static final int RIGHT_SOLENOID = 0;
+}
 }
```

Comment:
They are on the same side, think of a diffrent name.

Reply:
done.